syntax = "proto3";

package Telemetry;

message CsvParserConfig
{
    string subfolderName = 1;
    bool parseCan = 2;
    bool parseGps = 3;
    bool generateReport = 4;
}